src/main/resources/org/springframework/integration/config/xml/spring-integration-kafka-1.0.xsd

```diff
@@ -492,4 +492,182 @@
 		</xsd:complexType>
 	</xsd:element>
 
+	<xsd:element name="message-driven-channel-adapter">
+		<xsd:annotation>
+			<xsd:documentation>
+				The definition for the Spring Integration Kafka
+				KafkaMessageDrivenChannelAdapter.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:attributeGroup ref="integration:channelAdapterAttributes"/>
+			<xsd:attribute name="send-timeout" type="xsd:string">
+				<xsd:annotation>
+					<xsd:documentation><![CDATA[
+						Allows you to specify how long this inbound-channel-adapter
+						will wait for the message (containing the retrieved entities)
+						to be sent successfully to the message channel, before throwing
+						an exception.
+
+						Keep in mind that when sending to a DirectChannel, the
+						invocation will occur in the sender's thread so the failing
+						of the send operation may be caused by other components
+						further downstream. By default the Inbound Channel Adapter
```

Comment:
? The usual verbiage here is something like "Only applies if the MessageChannel can block - such as a bounded QueueChannel that is full".